src/main/webapp/app/exam/participate/summary/exercises/programming-exam-summary/programming-exam-summary.component.ts

```diff
@@ -39,6 +39,8 @@ export class ProgrammingExamSummaryComponent implements OnInit {
 
     @Input() isAfterResultsArePublished?: boolean = false;
 
+    @Input() instructorView?: boolean = false;
```

Comment:
I think we have decided to use new `input()` and `output()` decorators
```suggestion
    instructorView = input<boolean>(false);
```

Reply:
That's true, but for this I'd need to change the whole input chain which is quite big and way out for scope for this PR.